arch/arm/core/aarch64/vector_table.S

```diff
@@ -57,9 +57,8 @@ _ASM_FILE_PROLOGUE
  */
 
 	/* The whole table must be 2K aligned */
-	.align 11
```

Comment:
keeping it inside the section is the right thing to do.  keeping the directive out side also should have aligned the code following it ( I have not checked it) ? .  Otherwise I wonder how so far vector table was getting aligned just by chance.

Reply:
I have never seen any issue with the table alignment, but this is the second commit I see about this (see https://github.com/zephyrproject-rtos/zephyr/pull/30164/commits/97389981b30c9a29d396817de91b5a8f33bf34dc) so I think people saw some problems with this.

Reply:
Indeed. In my local, I have moved the vector table to other places, not in the beginning. Align 7 has issue.